.env.example

```diff
@@ -9,6 +9,7 @@ ADMIN_PASSWORD=admin
 
 LOG_CHANNEL=stack
 LOG_LEVEL=debug
+DIVISION_LOG_FILE_NAME=division.log
```

Comment:
Unfortunately, logging to a file does not help in a cluster deployment. The log would end up in some pod, which might also be destroyed over time, etc. Therefore, the feature should be removed or rebranded as a dev log (e.g. that only runs locally and via the feature flag). Especially as there is already an AuditLog for Prod about each event.

Reply:
Renamed to DIVISION_DEBUG_LOG with no set value